### PR TITLE
Add nodeSelector items. Fix #43

### LINKED ISSUE #43: Allow configuring nodeSelector
It would be nice to have the possibility to set additional nodeSelector filters. At least a global one common to the georchestra stack

### PR DESCRIPTION
**Pull request by jeanpommier**

### CODE REVIEW

**Comment by edevosc2c** [comment]
Didn't you miss the values in the values.yaml?

File: values.yaml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 georchestra:
+  # Set additional nodeSelector items
```
Comment (edevosc2c) [inline values.yaml:2]: I think you can set it like this instead:
```
# Set additional nodeSelector items
nodeSelector: {}
#  "georchestra": "true"
```
Which is more common for empty arrays by default.

Comment (jeanpommier) [reply values.yaml:2]: fine for me. Thanks

**Review by edevosc2c — APPROVED** [review]
Great for me, waiting on the CI to pass.

**Comment by jeanpommier** [comment]
> Great for me, waiting on the CI to pass.

Failed at `Run chart-testing (install)` stage, but this is not related to this PR, right ?

**Comment by edevosc2c** [comment]
> > Great for me, waiting on the CI to pass.
> 
> Failed at `Run chart-testing (install)` stage, but this is not related to this PR, right ?

It's github actions that is having some issues to handle the kubernetes cluster again :/. If the lint passed I think it's ok to merge.

**Pull request merged**